frontend/src/components/Navbar/linklist.ts

```diff
@@ -0,0 +1,22 @@
+export const linklist = [
```

Comment:
@Aksh-Bansal-dev why aren't you using the already created [navLink](https://github.com/BitByte-TPC/TPC-website/tree/main/frontend/src/components/navLinks) component?